schemas/oic.if.delete-response-schema.json

```diff
@@ -0,0 +1,24 @@
+{
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "description" : "Copyright (c) 2018 Open Connectivity Foundation, Inc. All rights reserved.",
+    "id": "http://openconnectivityfoundation.github.io/core/schemas/oic.if.delete-response-schema.json#",
+    "title": "Delete Response Format",
+    "definitions": {
+      "delete-response-schema": {
+        "type": "object",
+        "properties": {
+          "href": {
+            "type": "string",
+            "maxLength": 256,
+            "description": "The returned URI is relative to the device root path, i.e. starts with / ",
+            "format": "uri"
+          }
+        },
+        "additionalProperties": true,
```

Comment:
Is it possible to include additional Properties?  I thought only "href" was allowed.  What would it mean if other Properties were included in the DELETE response?